### DISCUSSION TITLE
Guests view: Add to dropdown list - ability to see different stats

### DISCUSSION THREAD

**Discussion by rondadams** [OpenRCT2/OpenRCT2#21476] [Feature requests]
The current Guests view only allows the ability to see Thoughts and Actions. 
Add to the dropdown selector: Cash in Pocket, Cash Spent, Number of Rides, Happiness, Energy, Hunger, Thirst, Nausea, etc. 
This would be helpful to see, especially the amount of money all your guests have.

**Comment by astraylife1** [comment]
I wish there was search bar as well so that you can sort it to just see guests "heading for x" ride or be able to type in a keyword and just have it filter the list
